templates/docs/examples/patterns/side-navigation/accordion.html

```diff
@@ -6,9 +6,11 @@
 {% block content %}
 {% include "docs/examples/patterns/side-navigation/_accordion.html" %}
 
+{% if not is_combined %}
```

Comment:
Does anything fail if we have the scripts?

I'd hope we can only remove scripts from combined if really necessary.

Reply:
@bartaz Including the scripts in combined view causes console errors (some scripts are only meant to be on certain accordion examples, and some can override each other). 

We can give this PR the same treatment as #5165 once it is merged (using a script block rather than an if-chech)

Reply:
Yes, let's wait for the scripts block.